plugins/BEdita/Core/src/Model/Entity/ObjectType.php

```diff
@@ -353,7 +436,26 @@ protected function _getSchema()
     }
 
     /**
-     * Check if an object type is child of another object type.
+     * Add internal associations as properties
+     *
+     * @return array
+     */
+    protected function associationProperties(): array
+    {
+        $assocProps = array_intersect(self::ASSOC_PROPERTIES, (array)$this->associations);
```

Comment:
Just a note. 
If the intention here is to possibly override `ASSOC_PROPERTIES` constant in an Entity that extends `ObjectEntity` we should take advantage of [late static bindings](https://www.php.net/manual/en/language.oop5.late-static-bindings.php) and use the `static` keyword instead of `self`.

Reply:
I don't think we will override this constant in an Entity... we will probably modify the constant as new `core` internal resources are added. Anyway it's safer to use `static` so I'm replacing it.